src/plopp/backends/matplotlib/image.py

```diff
@@ -232,7 +235,7 @@ def format_coord(self, x: float, y: float) -> str:
                     unit=self._hover_slicing['unit'][1],
                 ),
             ]
-            out += f', z={val.value:1.4f}'
+            out += f', z={scalar_to_string(val)}'
```

Comment:
I think `z` is confusing, especially since you could have a dimension labelled `z`. Can we indicate in some other way that this is the value?

Some ideas:
```
f(time=1 s, temp=1 K) = 100 counts
(time=1 s, temp=1K) -> 100 counts
```

Reply:
I went with `(time=1 s, temp=1 K) : 100 counts`